C7GameData/CombatResult.cs

```diff
@@ -0,0 +1,22 @@
+namespace C7GameData
+{
+	public enum CombatResult {
```

Comment:
Is it worth differentiating here between killed & captured, or whether the attacker advances? This seems adequate for now.